app/models/WorldpayTasks.class.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { getActiveCredential } = require('@utils/credentials')
+
+class WorldpayTasks {
```

Comment:
This probably doesn't need to be a class, if it just produces a list of tasks from a gateway account object it can just be a single function

Reply:
you could follow the pattern established for stripe tasks and create a utility function: https://github.com/alphagov/pay-selfservice/blob/master/app/utils/simplified-account/settings/stripe-details/tasks.js

Reply:
The Stripe tasks are more complex and their state is stored in connector db, so I imagine that's why they'd be handled differently. For me the question to ask when reviewing is the code readable? I think what I've got here currently is easily understandable. We can refactor it to be more like the Stripe tasks pattern if we need be (for example when we come to add more Worldpay tasks). Trying to make this more like Stripe tasks is going to hurt my head at the moment.